docs/README.md

```diff
@@ -6,9 +6,16 @@ Event-driven architecture enables systems to share data changes in real time acr
 
 An Event Bus allows a system to publish a stream of changes to its data. Systems that are interested in those data changes can receive the information in real time and use it to send updates to trigger actions, send updates to Veterans, communicate with other internal teams, and more.
 
```

Comment:
Can we add a sentence here that explains what this video is? Maybe something like "Watch this video to learn more about how Event Bus works."